### DISCUSSION TITLE
Enhancement: Passkey/Passwordless Support

### DISCUSSION THREAD

**Discussion by th3raid0r** [danny-avila/LibreChat#2683] [Feature Requests & Suggestions]
As an admin, I know many users are going to be rocking with `hunter2` or it's equivalent - passkeys would really help with keeping the lazy/low-effort users safe.

Currently I can simply allow only 3rd party/social logins that support this - but that overhead is large-ish for my use-case of a server for only friends and family.

**Comment by bpawnzZ** [comment]
I second this. I have my instance protected behind Cloudflare zero trust. I am confident in my security. While I am already logged in 90% of the time. I could be done with the login as well
